api/Dockerfile

```diff
@@ -22,6 +32,8 @@ FROM alpine:3.13
 ENV TURING_PORT "8080"
 ENV TURING_USER "turing"
 ENV TURING_USER_GROUP "app"
+ENV TURING_OPENAPI_BUNDLE_FILE "/app/openapi.bundle.yaml"
+ENV OPENAPICONFIG_OPENAPIBUNDLEFILENAME "openapi.bundle.yaml"
```

Comment:
Nit: shall you re-use `openapi.bundle.yaml` value via docker build arg or env var? It's also hardcoded at Line 7. Would be nice to define it only once in case if we will ever need to change it